gpt4all-backend/gptj.cpp

```diff
@@ -26,9 +27,16 @@
 #include <sstream>
 #include <unordered_set>
 
-// default hparams (GPT-J 6B)
-static const size_t MB = 1024*1024;
 
+namespace {
+const char *modelType_ = "MPT";
+
+constexpr inline unsigned long long operator ""_MiB(unsigned long long bytes) {
```

Comment:
These are good changes, but I'd prefer them as sep PR

Reply:
The `modelType_` this is required, but I could remove the _MiB operator